### PR TITLE
done.

### PR DESCRIPTION
**Pull request by shahad197**

### CODE REVIEW

**Comment by shuruqt1000** [comment]
please re-do your comments you didn't explain (what the code dose!) not what you wrote

**Review by shuruqt1000 — CHANGES_REQUESTED** [review]
explain you code explain what each function dose and how you pass values if any and why

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml [hunk 1]
```diff
@@ -1,12 +1,52 @@
...
+open class Shape() {             // i open a class it name (shape)
+    val shapeName = "circlr"     // the properties
+    val shapeColor = "blue"
+    fun info(){                            // i put a function 'info'
+        println("$shapeName,$shapeColor")
+    }
+  open  fun computeArea():Double{            // i put another function 'computArea and i make it 'Double
+        var area = 0.0
+        return area
+    }
```
Comment (shuruqt1000) [inline HW-Week2_Day03_Inheritance_and_ClassTypes.iml:27]: why are these open functions ?

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml [hunk 2]
```diff
@@ -1,12 +1,52 @@
...
+}
+class Triangle:Shape(){
+    val base = 10.5
+    val height = 12.8
+    override fun computeArea(): Double {
+        return (base*height)/2                   /// this is the three sub class amd a made a calculation
```
Comment (shuruqt1000) [inline HW-Week2_Day03_Inheritance_and_ClassTypes.iml:33]: why did you do override to this ?

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml [hunk 3]
```diff
@@ -1,12 +1,52 @@
...
-    <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="KotlinJavaRuntime" level="project" />
-  </component>
-</module>
\ No newline at end of file
+fun main(){
+
+val triangle = Triangle()                     // i defined the x value
+    println(triangle.computeArea())           // i print the function x from x class
+
```
Comment (shuruqt1000) [inline HW-Week2_Day03_Inheritance_and_ClassTypes.iml:5]: what do you mean by value X 
what's the function X ?

**Review by shuruqt1000 — CHANGES_REQUESTED** [review]

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml (hunk 1 above)
Comment (shahad197) [reply HW-Week2_Day03_Inheritance_and_ClassTypes.iml:27]: عشان نورثها للسب كلاسس

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml (hunk 2 above)
Comment (shahad197) [reply HW-Week2_Day03_Inheritance_and_ClassTypes.iml:33]: عشان قانون المساحة يختلف من شكل الى شكل اخر

File: HW-Week2_Day03_Inheritance_and_ClassTypes.iml (hunk 3 above)
Comment (shahad197) [reply HW-Week2_Day03_Inheritance_and_ClassTypes.iml:5]: i print the function  computeArea from Triangle  class

**Comment by shuruqt1000** [comment]
completed

**Pull request closed without merging**